admin-tools/src/webapp/vm/sites/chef_sites_edit_tool.vm

```diff
@@ -1,4 +1,7 @@
 ##<!-- $Header: /cvs/sakai2/legacy/tools/src/webapp/vm/admin/chef_sites_edit_tool.vm,v 1.3 2005/05/19 19:53:01 gsilver.umich.edu Exp $ -->
+<script>
+    includeLatestJQuery('chef_sites_edit_tool.vm');
```

Comment:
Can this be moved to the bottom of the page as it would be ideal?

Reply:
Done